tools/test-backend

```diff
@@ -66,6 +67,56 @@ if __name__ == "__main__":
                       help="Show which tests are slowest.")
 
     (options, args) = parser.parse_args()
+
+    thedir = 'zerver/tests/'
+
+    for suite in args:
+        args[args.index(suite)]=suite.replace("/", ".")
```

Comment:
In the Zulip Python style, we use `foo = bar` for the whitespace; can you tweak this PR to match that?